pkg/steps/release/promote_test.go

```diff
@@ -578,8 +578,8 @@ func TestGetImageMirror(t *testing.T) {
 
 	for _, testCase := range testCases {
 		t.Run(testCase.name, func(t *testing.T) {
-			if actual, expected := getImageMirrorTarget(testCase.tags, testCase.pipeline, "registry.ci.openshift.org"), testCase.expected; !reflect.DeepEqual(actual, expected) {
-				t.Errorf("%s: got incorrect ImageMirror mapping: %v", testCase.name, diff.ObjectDiff(actual, expected))
+			if actual, _ := getImageMirrorTarget(testCase.tags, testCase.pipeline, "registry.ci.openshift.org"); !reflect.DeepEqual(actual, testCase.expected) {
```

Comment:
Why?

Reply:
Bcause `getImageMirrorTarget` now has two return values, and golang doesn't allow sth like

```
if a, b, c := funcWithTworeturnvals, c; {...
```

Reply:
Sure, but what was wrong about the previous code ... ?

Reply:
It doesn't compile because `getImageMirrorTarget` now has two return values